responsibleai/tests/test_model_analysis.py

```diff
@@ -252,6 +261,36 @@ def run_model_analysis(model, train_data, test_data, target_column,
             validate_explainer(model_analysis, train_data, test_data, classes)
 
 
+def validate_state_directory(path, manager_type):
+    all_dirs = os.listdir(path)
+    assert manager_type in all_dirs
+    all_component_paths = os.listdir(path / manager_type)
+    for component_path in all_component_paths:
+        # TODO: Add code to check if the component_path is GUID
+        dm = DirectoryManager(path / manager_type, component_path)
+
+        config_path = dm.get_config_directory()
+        data_path = dm.get_data_directory()
+        explainer_path = dm.get_explainer_directory()
+
+        if manager_type == ManagerNames.EXPLAINER:
+            assert not config_path.exists()
+            assert data_path.exists()
+            assert not explainer_path.exists()
```

Comment:
It seems none of them are using the explainer_path right now?  Perhaps at this point it shouldn't exist then?

Reply:
I plan to move causal to explainer path. The causal is state management is a little bit tricky. So keeping it in data directory for now. Will address that in up coming PR.